pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -268,6 +271,16 @@ func (c ClusterSpec) IsKubernetesDashboardEnabled() bool {
 	return c.KubernetesDashboard == nil || c.KubernetesDashboard.Enabled
 }
 
+// KubeLB contains settings for the kubeLB component as part of the cluster control plane. This component is responsible for managing load balancers.
+type KubeLB struct {
+	// Controls whether kubeLB is deployed or not.
+	Enabled bool `json:"enabled,omitempty"`
```

Comment:
If this is omitempty, then enabled=false will be stripped from the resource. Won't this make it impossible to disable KubeLB?

Reply:
In that case, the kubeLB struct will be empty which is interpreted as kubeLB being disabled. https://github.com/kubermatic/kubermatic/blob/dd0fda7d4abc1e36d1b63409f886dd7595bf9fbc/pkg/apis/kubermatic/v1/cluster.go#L281

Reply:
^ Wait, that's the reverse of what we want. It's enabled by default in this case. I'll fix this.

Reply:
Should be fixed now 🤙

Reply:
This is due to be fixed in a follow-up PR. I misunderstood what you asked for :D